language-support/java/codegen/BUILD.bazel

```diff
@@ -211,6 +211,17 @@ daml_compile(
     target = "1.11",
 )
 
+daml_compile(
+    name = "integration-tests-model-1_12",
+    srcs = glob(
+        [
+            "src/it/daml/Tests/*.daml",
+            "src/it/daml/Tests/Lib1_112.daml",
```

Comment:
I'm fairly sure this line doesn't point to anything (there's no such file) and just gets ignored by glob, but it's just like the ones above. I'll leave it to a separate PR to clean this up.